modules/@ergonode/core/src/plugins/core.js

```diff
@@ -40,4 +42,47 @@ export default ({
             .map(setOrder)
             .sort(sortByOrder);
     });
+    inject('extendedProps', async ({
+        key, name = null,
+    }) => {
+        let componentProps = {};
+        const extendedProps = await app.$getExtendMethod(key, {
+            $this: app,
+        });
+
+        if (extendedProps.length) {
+            extendedProps.forEach((props) => {
+                componentProps = deepmerge(componentProps, props);
+            });
+        }
+        if (name && !isEmpty(componentProps)) {
```

Comment:
Consider returning an empty object before everything is happening. In case there it's not defined, no need to load extendedProps